[2025-10-11T04:45:02.884+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-11T04:45:02.930+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline_flow.check_create_stg_tables scheduled__2025-09-28T00:00:00+00:00 [queued]>
[2025-10-11T04:45:02.940+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline_flow.check_create_stg_tables scheduled__2025-09-28T00:00:00+00:00 [queued]>
[2025-10-11T04:45:02.942+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-11T04:45:02.957+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): check_create_stg_tables> on 2025-09-28 00:00:00+00:00
[2025-10-11T04:45:02.972+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=7375) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-11T04:45:02.971+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pipeline_flow', 'check_create_stg_tables', 'scheduled__2025-09-28T00:00:00+00:00', '--job-id', '82', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_dag.py', '--cfg-path', '/tmp/tmp0oadeaol']
[2025-10-11T04:45:02.974+0000] {standard_task_runner.py:64} INFO - Started process 7384 to run task
[2025-10-11T04:45:02.974+0000] {standard_task_runner.py:91} INFO - Job 82: Subtask check_create_stg_tables
[2025-10-11T04:45:03.044+0000] {task_command.py:426} INFO - Running <TaskInstance: pipeline_flow.check_create_stg_tables scheduled__2025-09-28T00:00:00+00:00 [running]> on host 321597a50810
[2025-10-11T04:45:03.178+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='oluwaseyiogundimu10@gmail.com' AIRFLOW_CTX_DAG_OWNER='seyi' AIRFLOW_CTX_DAG_ID='pipeline_flow' AIRFLOW_CTX_TASK_ID='check_create_stg_tables' AIRFLOW_CTX_EXECUTION_DATE='2025-09-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-28T00:00:00+00:00'
[2025-10-11T04:45:03.180+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-11T04:45:03.198+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T04:45:03.283+0000] {connection.py:43} INFO - Table 'stg_users' in Postgres.
[2025-10-11T04:45:03.285+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T04:45:03.379+0000] {connection.py:43} INFO - Table 'users' in Postgres.
[2025-10-11T04:45:03.380+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T04:45:03.458+0000] {connection.py:43} INFO - Table 'stg_movies' in Postgres.
[2025-10-11T04:45:03.461+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T04:45:03.551+0000] {connection.py:43} INFO - Table 'movies' in Postgres.
[2025-10-11T04:45:03.553+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T04:45:03.642+0000] {connection.py:43} INFO - Table 'stg_ratings' in Postgres.
[2025-10-11T04:45:03.643+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T04:45:03.730+0000] {connection.py:43} INFO - Table 'ratings' in Postgres.
[2025-10-11T04:45:03.731+0000] {load.py:32} INFO - Tables exist.
[2025-10-11T04:45:03.732+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-10-11T04:45:03.733+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-11T04:45:03.754+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pipeline_flow, task_id=check_create_stg_tables, run_id=scheduled__2025-09-28T00:00:00+00:00, execution_date=20250928T000000, start_date=20251011T044502, end_date=20251011T044503
[2025-10-11T04:45:03.833+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-10-11T04:45:03.887+0000] {taskinstance.py:3503} INFO - 3 downstream tasks scheduled from follow-on schedule check
[2025-10-11T04:45:03.891+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
